### PR TITLE
Move butter knife and renderers to the sample

### LINKED ISSUE #70: Remove ButterKnife and Renderers library.
In order to reduce the number of dependencies of this framework I suggest to remove ButterKnife and Renderers libraries. @Serchinastico, do you agree? @luis-ibanez instead of update ButterKnife we should delete it and keep the code related to this library and the renderers library just in the example project.

### PR DESCRIPTION
**Pull request by luis-ibanez**
Fixes #70

### CODE REVIEW

**Comment by pedrovgs** [comment]
Thanks @luis-ibanez the repository is not passing in Travis-CI due to compile errors.

**Comment by pedrovgs** [comment]
@luis-ibanez can you review the build output? The code is not compiling!

**Comment by pedrovgs** [comment]
Thank you so much! 👍

**Comment by pedrovgs** [comment]
Thank you so much! 👍

**Comment by pedrovgs** [comment]
@Serchinastico can you review this PR?

File: rosie/src/main/java/com/karumi/rosie/view/RosieFragment.java [hunk 1]
```diff
@@ -59,9 +59,7 @@
       Bundle savedInstanceState) {
     injectDependencies();
     int layoutId = getLayoutId();
-    View view = inflater.inflate(layoutId, container, false);
-    ButterKnife.bind(this, view);
-    return view;
+    return inflater.inflate(layoutId, container, false);
```
Comment (Serchinastico) [inline rosie/src/main/java/com/karumi/rosie/view/RosieFragment.java]: To be consistent I'd add an `onPrepareFragment` method to let clients bind their views from their fragments as well. Include the view as a parameter.

**Comment by Serchinastico** [comment]
Besides the fragment method, it LGTM

File: rosie/src/main/java/com/karumi/rosie/view/RosieFragment.java (hunk 1 above)
Comment (luis-ibanez) [inline rosie/src/main/java/com/karumi/rosie/view/RosieFragment.java]: I was told no to do that and initialise it in onCreateView of the Fragment, I really don't have any preference in any of those solutions. @pedrovgs ?

Comment (pedrovgs) [inline rosie/src/main/java/com/karumi/rosie/view/RosieFragment.java]: I think the current implementation is good enough :) IMHO we can already merge this PR :)

**Comment by luis-ibanez** [comment]
Until we decide what to do with the Fragments, I cleaned a little bit the PR

**Pull request merged**